packages/@uppy/companion/src/server/controllers/callback.js

```diff
@@ -4,6 +4,18 @@
 const tokenService = require('../helpers/jwt')
 const logger = require('../logger')
 
+const closePageHtml = `
+  <!DOCTYPE html>
+  <html>
+  <head>
+      <meta charset="utf-8" />
+      <script>
+        window.close()
+      </script>
+  </head>
+  <body></body>
```

Comment:
IMHO it would be worth displaying something in the body in case the browser refuses to close the window:
```suggestion
  <body>Authentication failed.</body>
```

Reply:
Should we i18n this message?

Reply:
If we have access to the user's preferred language at this point, sure why not. We should keep in mind that unless the browser does something weird, users would never see this message so it may not be worth it.

Reply:
Yea I agree. I don't see any other i18n code in companion.

Reply:
BTW in the success scenario, maybe we should do the same.